extensions/amp-story/1.0/amp-story-interactive-poll.js

```diff
@@ -201,7 +205,30 @@ export class AmpStoryInteractivePoll extends AmpStoryInteractive {
       currOption.querySelector(
         '.i-amphtml-story-interactive-option-percentage-text'
       ).textContent = `${percentage}%`;
-      currOption.setAttribute('style', `flex-grow: ${percentage} !important`);
+
+      currOption
+        .querySelector('.i-amphtml-story-interactive-option-percent-bar')
+        .setAttribute(
+          'style',
+          `transform: scaleX(${percentage * 0.01 * 2}) !important`
+        );
+
+      const textContainer = currOption.querySelector(
+        '.i-amphtml-story-interactive-option-text-container'
+      );
+
+      textContainer.setAttribute(
+        'style',
+        `transform: translateX(${
+          percentage > 25
```

Comment:
I'm trying to follow the logic here, if the translation is percentage-50 and MIN_PADDING is 20, shouldn't the limit percentage here be 30? So that percentage-50=20